stacks/its.js

```diff
@@ -0,0 +1,129 @@
+                Cl.stringAscii(AXELAR_HUB_IDENTIFIER),
+            ],
+            senderKey: privateKey,
+            network: networkType,
+            postConditionMode: PostConditionMode.Allow,
+            anchorMode: AnchorMode.Any,
+            fee: 10_000,
+        });
+        const result = await broadcastTransaction({
+            transaction: registerTransaction,
+            network: networkType,
+        });
+
+        printInfo(`Finished setting ${trustedChain} as trusted`, result.txid);
+
+        // Wait a bit before executing next transaction
+        await new Promise((resolve) => setTimeout(resolve, 5_000));
+    }
+}
+
+async function removeTrustedAddress(privateKey, networkType, config, chain, args) {
+    const contracts = chain.contracts;
+    if (!contracts.InterchainTokenService?.address) {
+        throw new Error(`Contract InterchainTokenService not yet deployed`);
+    }
+    if (!contracts.InterchainTokenServiceImpl?.address) {
+        throw new Error(`Contract InterchainTokenServiceImpl not yet deployed`);
+    }
+
+    const trustedChains = parseTrustedChains(config.chains, args);
+
+    for (const trustedChain of trustedChains) {
+        printInfo(`Removing trusted address for ${trustedChain}`);
+
+        const itsAddress = contracts.InterchainTokenService.address.split('.');
+        const registerTransaction = await makeContractCall({
+            contractAddress: itsAddress[0],
+            contractName: itsAddress[1],
+            functionName: 'remove-trusted-address',
+            functionArgs: [Cl.address(contracts.InterchainTokenServiceImpl.address), Cl.stringAscii(trustedChain)],
+            senderKey: privateKey,
+            network: networkType,
+            postConditionMode: PostConditionMode.Allow,
+            anchorMode: AnchorMode.Any,
+            fee: 10_000,
+        });
+        const result = await broadcastTransaction({
+            transaction: registerTransaction,
+            network: networkType,
+        });
+
+        printInfo(`Finished removing ${trustedChain} from trusted`, result.txid);
+
+        // Wait a bit before executing next transaction
+        await new Promise((resolve) => setTimeout(resolve, 5_000));
+    }
+}
+
+async function processCommand(command, config, chain, args, options) {
+    const { privateKey, networkType } = await getWallet(chain, options);
+
+    await command(privateKey, networkType, config, chain, args, options);
+}
+
+async function mainProcessor(command, options, args, processor) {
+    const config = loadConfig(options.env);
+    const chain = getChainConfig(config.chains, options.chainName);
+    await processor(command, config, chain, args, options);
+    saveConfig(config, options.env);
+}
+
+if (require.main === module) {
+    const program = new Command();
+    program.name('Gas Service Commands').description('Stacks GasService scripts');
+
+    const setTrustedAddressCmd = new Command()
+        .name('set-trusted-address')
```

Comment:
```suggestion
        .name('set-trusted-chain')
```
since we are not allowing addresses to be configured in this cmd